glib/src/variant.rs

```diff
@@ -290,6 +290,28 @@ impl Variant {
         }
     }
 
+    /// Tries to extract a `&[u8]` from a variant of type `ay`.
```

Comment:
What does `ay` mean?

Reply:
https://gitlab.gnome.org/GNOME/glib/-/blob/be4b8cbe96ecd8d7d738fb9c80729285123ecdf3/glib/gvarianttype.c#L125

Reply:
Maybe write it in full text in the documentation then. Not everyone is a gnome developer (myself included).

Reply:
Done.